api/src/controllers/users.ts

```diff
@@ -142,4 +147,92 @@ export default class UsersController {
 
         return res.status(204).json({ msg: "Logged out successfully." });
     }
+
+    public async getFriends(req: Request, res: Response) {
+        const userRepository = this.repository;
```

Comment:
Improve the signature for this call, I was thinking of maybe getting the current user from the JWT.